content-service/src/main/java/io/meeds/news/service/impl/NewsServiceImpl.java

```diff
@@ -1018,6 +1037,9 @@ private List<News> buildDraftArticles(NewsFilter filter, Identity currentIdentit
                                                 // block
                                                 e.printStackTrace();
```

Comment:
remove the e.printStackTrace(); and throw a specific exception to the rest layer instead

Reply:
the method returns null if an exception is thrown , i have added a log in the catch block!